wqflask/wqflask/correlation_matrix/show_corr_matrix.py

```diff
@@ -19,28 +19,31 @@
 # This module is used by GeneNetwork project (www.genenetwork.org)
 
 import datetime
-import math
 import random
 import string
 
-import rpy2.robjects as ro
-from rpy2.robjects.packages import importr
 
 import numpy as np
 import scipy
 
 from base import data_set
 from base.webqtlConfig import GENERATED_TEXT_DIR
-from functools import reduce
-from functools import cmp_to_key
-from utility import webqtlUtil
 from utility import helper_functions
 from utility import corr_result_helpers
 from utility.redis_tools import get_redis_conn
 
+
+from gn3.computations.principal_component_analysis import compute_pca
+
+from gn3.computations.principal_component_analysis import process_factor_loadings_tdata
+from gn3.computations.principal_component_analysis import generate_pca_traits_vals
+from gn3.computations.principal_component_analysis import generate_pca_temp_dataset
```

Comment:
This is minor, but it might be better to call this "generate_pca_temp_traits" (since it's not really a "dataset" in the way the term is used elsewhere in GN)